Makefile.toml

```diff
@@ -0,0 +1,68 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+TARGET = { value = "${CARGO_MAKE_RUST_TARGET_TRIPLE}", condition = { env_not_set = [
+    "TARGET",
+] } }
+
+[tasks.format]
+install_crate = { crate_name = "rustfmt-nightly", binary = "rustfmt", test_arg = "--help" }
+command = "cargo"
+args = ["+nightly", "fmt", "--", "--check"]
+
+[tasks.clippy]
+install_crate = "cargo-clippy"
+command = "cargo"
+args = ["clippy"]
```

Comment:
I would add `--all-targets --all-features` to reduce the difference with what is done on the CI workflow.